ci/minimum-tests.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - xarray>=0.16.1
   # xhistogram 0.1.3 introduced an error that broke xskillscore
   # see https://github.com/xgcm/xhistogram/issues/48
-  - xhistogram!=0.1.3
+  - xhistogram==0.1.2
```

Comment:
@dougiesquire released xh=0.2.0

Reply:
yeah fails on xh==0.1.2 as well as xh latest. Guessing it's an xarray/dask/numpy update

Reply:
yeah no worries mate. I created an issue for it here https://github.com/xarray-contrib/xskillscore/issues/311